tests/x11/gnucash.pm

```diff
@@ -22,19 +22,18 @@ sub run() {
     # needed for viewing
     ensure_installed("yelp");
     x11_start_program("gnucash");
-    assert_screen 'test-gnucash-1', 3;
+    assert_screen 'test-gnucash-1';
     send_key "ctrl-h";    # open user tutorial
-    wait_idle 5;
-    assert_screen 'test-gnucash-2', 3;
-    send_key "alt-f4";    # Leave tutorial window
-                          # Leave tips windows for GNOME case
-    if (check_var("DESKTOP", "gnome") || get_var("DESKTOP") eq "xfce") { sleep 3; send_key "alt-c"; }
-    wait_idle;
-    send_key "ctrl-q";    # Exit
-
-    if (check_screen("gnucash-save-changes", 10)) {
-        send_key "alt-w";    # Close _Without Saving
+    assert_screen 'test-gnucash-2';
+    # Leave tutorial window
+    wait_screen_change { send_key 'alt-f4' };
```

Comment:
it's the other way around: 'alt-f4' is sent, then os-autoinst should wait for screen change